sql/moz-fx-data-shared-prod/mozilla_org/gclid_conversions/metadata.yaml

```diff
@@ -0,0 +1,13 @@
+friendly_name: Gclid Conversions V1
+description: |-
+  1-row per conversion. Provides conversion events to
+  send to Google Ads.
+owners:
+- frank@mozilla.com
+labels: {}
+bigquery: null
+references: {}
```

Comment:
Interestingly it looks like specifying this value explicitly overrides the generated value: https://github.com/mozilla/private-bigquery-etl/blob/130e802354f72f7861bdbd71b10f8e489615cb46/sql/moz-fx-data-shared-prod/mozilla_org/gclid_conversions/metadata.yaml#L14 (see also the [diff](https://github.com/mozilla/private-bigquery-etl/commit/130e802354f72f7861bdbd71b10f8e489615cb46)). We probably want to avoid that by removing this line or always overriding `references` in generation logic.